whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/acl/AutomaticPermanentBlocks.java

```diff
@@ -36,7 +36,7 @@ public AutomaticPermanentBlocks(final DateTimeProvider dateTimeProvider,
     }
 
     @Override
-    @Scheduled(cron = "0 0 0 * * *")
+    @Scheduled(cron = "0 0 0 * * *", zone = EUROPE_AMSTERDAM)
```

Comment:
Does this job need to run on Amsterdam time? the ACL data will be in UTC

Reply:
probably best to have it run in UTC. Updated to this effect